wrappers/here.rb

```diff
@@ -457,10 +454,13 @@ def here_strict_restriction(param_value)
     end
 
     ##
-    # dist: value in meters
-    # 100km: 7, 1200km: 2, 1400km: 1
-    def coef_distance(dist)
-      7 - [(dist.ceil / 1000) / 201, 6.0].min
+    # distance > 2000km : 5
```

Comment:
Add more infos to explain what is the expected result and why

Reply:
I moved the explanation to the method.